### ISSUE TITLE
Glorious Model O not seen

### ISSUE THREAD

**Issue by Cry92** [libratbag/libratbag#1423]
**Information**

- `ratbagd` version (`ratbagd --version`): 0.17
- Distribution: Nobara 37
- Kernel version (ex. `uname -srmo`): Linux 6.1.9-200.fsync.fc37.x86_64 x86_64 GNU/Linux
Device info (if applicable): 
```
$ ratbagctl <device> info
Unable to find device glorious-model-o.device
```

**Describe the bug**
Glorious Model o is detected and not detected at the same time. Cant pull info on it but can see it in the dump. Piper does not see the device at all.

**To Reproduce**
Steps I have taken:

ratbagctl list
No devices available.

su -c 'ratbagd --verbose=raw'
Initializing libratbag
ratbag debug: New device: ITE Tech. Inc. ITE Device
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 048d:5702
ratbag debug: New device: Massdrop Inc. ALT Keyboard
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 04d8:eed3
ratbag debug: New device: Massdrop Inc. ALT Keyboard
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 04d8:eed3
ratbag debug: New device: Massdrop Inc. ALT Keyboard
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 04d8:eed3
ratbag debug: New device: Glorious Model O Wireless
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 258a:2022
ratbag debug: New device: Glorious Model O Wireless
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 258a:2022
ratbag debug: New device: Glorious Model O Wireless
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 258a:2022
DBus server ready

Added in the device number into /usr/share/libratbag/glorious-model-o.device
Then repeated ratbagd --verbose=raw (below)

ratbagctl list
No devices available.

**Logs**
Please start the daemon with verbosity and reproduce the issue.

First make sure it isn't running already.
```
$ ratbagd --verbose=raw
Initializing libratbag
ratbag debug: New device: ITE Tech. Inc. ITE Device
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 048d:5702
ratbag debug: New device: Massdrop Inc. ALT Keyboard
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 04d8:eed3
ratbag debug: New device: Massdrop Inc. ALT Keyboard
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 04d8:eed3
ratbag debug: New device: Massdrop Inc. ALT Keyboard
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: No data file found for 04d8:eed3
ratbag debug: New device: Glorious Model O Wireless
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: device assigned driver sinowealth
ratbag debug: hidraw info: bus 0x03 vendor 0x258a product 0x2022
ratbag debug: Glorious Model O Wireless is device '/dev/hidraw1'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: hidraw info: bus 0x03 vendor 0x258a product 0x2022
ratbag debug: Glorious Model O Wireless is device '/dev/hidraw2'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: - HID report ID 01
ratbag debug: - HID report ID 02
ratbag debug: - HID report ID 03
ratbag debug: - HID report ID 04
ratbag debug: - HID report ID 05
ratbag raw: feature set:   05 02 01 00 00 00
ratbag raw: feature set:   05 01 00 00 00 00
ratbag raw: feature get:   05 01 00 00 00 00
ratbag info: firmware version: 
ratbag raw: feature set:   05 11 00 00 00 00
ratbag raw: feature get:   05 11 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
ratbag error: Could not read device configuration: 65
ratbag debug: New device: Glorious Model O Wireless
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: device assigned driver sinowealth
ratbag debug: hidraw info: bus 0x03 vendor 0x258a product 0x2022
ratbag debug: Glorious Model O Wireless is device '/dev/hidraw1'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: hidraw info: bus 0x03 vendor 0x258a product 0x2022
ratbag debug: Glorious Model O Wireless is device '/dev/hidraw2'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: - HID report ID 01
ratbag debug: - HID report ID 02
ratbag debug: - HID report ID 03
ratbag debug: - HID report ID 04
ratbag debug: - HID report ID 05
ratbag raw: feature set:   05 02 01 00 00 00
ratbag raw: feature set:   05 01 00 00 00 00
ratbag raw: feature get:   05 01 00 00 00 00
ratbag info: firmware version: 
ratbag raw: feature set:   05 11 00 00 00 00
ratbag raw: feature get:   05 11 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
ratbag error: Could not read device configuration: 65
ratbag debug: New device: Glorious Model O Wireless
ratbag debug: Using data directory '/usr/share/libratbag'
ratbag debug: device assigned driver sinowealth
ratbag debug: hidraw info: bus 0x03 vendor 0x258a product 0x2022
ratbag debug: Glorious Model O Wireless is device '/dev/hidraw1'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: hidraw info: bus 0x03 vendor 0x258a product 0x2022
ratbag debug: Glorious Model O Wireless is device '/dev/hidraw2'.
ratbag debug: Parsing HID report descriptor
ratbag debug: Parsing HID report descriptor
ratbag debug: - HID report ID 01
ratbag debug: - HID report ID 02
ratbag debug: - HID report ID 03
ratbag debug: - HID report ID 04
ratbag debug: - HID report ID 05
ratbag raw: feature set:   05 02 01 00 00 00
ratbag raw: feature set:   05 01 00 00 00 00
ratbag raw: feature get:   05 01 00 00 00 00
ratbag info: firmware version: 
ratbag raw: feature set:   05 11 00 00 00 00
ratbag raw: feature get:   05 11 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00
ratbag error: Could not read device configuration: 65
DBus server ready

```

**Comment by staticssleever668** [comment]
This is Model O Wireless, not Model O, and they are completely different under the hood.

Duplicate: #1172.
TLDR: currently we don't support the wireless version, however, there is another application you can use: https://github.com/korkje/mow (or updated fork https://github.com/dxbednarczyk/mxw).
